tensorrec/tensorrec.py

```diff
@@ -56,7 +56,13 @@ def __init__(self,
         If True, a bias value will be calculated for every user feature and item feature.
         """
 
-        # Arg-check
+	# check TensorFlow version
+        major,minor,patch = tf.__version__.split(".")
+        if int(major) < 1 or int(major) == 1 and int(minor) < 7:
+            raise RuntimeError("""You need to have at least TensorFlow version 1.4 installed in order to use 
```

Comment:
Shouldn't the string say `version 1.7`?

Reply:
sorry my bad! I made the fix for version 1.4 and then when I realized that 1.7. is needed I forgot to update the string :( 
I will also fix the intendations. sorry for bad patch

Reply:
Ah I just saw your Issue re: version 1.4 adding the `data` module. 

We also use `Iterator`s whose attributes changed in 1.7, so anything under 1.7 will break.

Reply:
well as u see in the if statement I ask for 1.7 it is just the string that was still 1.4. 
but what about the whitespace? my editor shows me the whitespace as spaces not tabs. but CI seems to fail. I will send another pullrequest in a minute.